### PR TITLE
Add Doctrine icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/8572

**Popularity metric:**

The Similarweb rank is .467,645. See https://www.similarweb.com/website/doctrine-project.org

Also, it has ~9.8k stars on GitHub. See https://github.com/doctrine/orm

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Resized, unified paths, optimized.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4316,6 +4316,11 @@
             "hex": "2ECE53",
             "source": "https://github.com/docsifyjs/docsify/blob/d01841fd9d829adeeae6cde1d5818ce8798c7e58/docs/_media/icon.svg"
         },
+        {
+            "title": "Doctrine",
+            "hex": "FA6A3C",
+            "source": "https://www.doctrine-project.org/logos/doctrine-logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:4322]: We don't need to link to the specific file if it's not on a repository - more likely to 404 if they rebrand!

```suggestion
            "source": "https://www.doctrine-project.org"
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4316,6 +4316,11 @@
             "hex": "2ECE53",
             "source": "https://github.com/docsifyjs/docsify/blob/d01841fd9d829adeeae6cde1d5818ce8798c7e58/docs/_media/icon.svg"
         },
+        {
+            "title": "Doctrine",
+            "hex": "FA6A3C",
```
Comment (adamrusted) [inline _data/simple-icons.json:4321]:
```suggestion
            "hex": "FC6A31",
```

I'm getting the above from my editor. Where did you find the other one?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
Icon itself is a perfect match, btw. Just a couple queries on the JSON.

File: _data/simple-icons.json (hunk 2 above)
Comment (CodeShakingSheep) [reply _data/simple-icons.json:4321]: Honestly, I don't recall but I just checked the color in the SVG and it matches your color suggestion. So, I will make a commit using that color.

**Pull request merged**